doc/containers.md

```diff
@@ -357,6 +358,28 @@ uid         | int       | 0                 | no        | UID of the device owne
 gid         | int       | 0                 | no        | GID of the device owner in the container
 mode        | int       | 0660              | no        | Mode of the device in the container
 
+### Type: proxy
+Proxy devices are processes that forward data to/from ports in the container and
+ports in host. Used to communicate easily between two network namespaces.
+
+There will eventually be three supported connection types:
+ - `TCP`
+ - `UDP`
+ - `Unix`
```

Comment:
We don't document what doesn't exist yet. I'd just remove that part and then file a Github issue to track the addition of the missing types.